load-test-framework/src/main/java/com/google/pubsub/clients/remote/RemoteTask.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+package com.google.pubsub.clients.remote;
+
+import com.google.pubsub.clients.common.LoadTestRunner;
+import com.google.pubsub.clients.common.MetricsHandler;
+import com.google.pubsub.clients.common.Task;
+import com.google.pubsub.flic.common.LoadtestGrpc;
+import com.google.pubsub.flic.common.LoadtestProto;
+import io.grpc.ManagedChannelBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+
+/**
+ * Runs a task that executes command on the remote server.
+ */
+class RemoteTask extends Task {
```

Comment:
Given that the loadtest framework already consists of code that runs locally (Driver) and code that runs remotely ("Client", which runs in VMs), the term remote here is confusing. I think this should be called AdapterTask -- it adapts various tasks (in other languages) to work with the primary Java framework. The directory should also be renamed "remote" to "adapter", although I don't actually see the need for this to be in its own sub-package.